src/build-tx/request.js

```diff
@@ -45,6 +45,7 @@ export type Request = {
     floorBaseFee?: boolean; // DOGE floor base fee to the nearest integer
     dustOutputFee?: number; // DOGE fee for every output below dust limit
     skipUtxoSelection?: boolean; // use custom utxo selection, without algorithm
+    skipPermutation?: boolean; // RBF: do not sort inputs/outputs. use exact requested order
```

Comment:
nitpick, optional: It's not very clear to me, how am I supposed to "request" an exact order. I think this might be a slightly better wording:
```suggestion
    skipPermutation?: boolean; // Do not sort inputs/outputs and preserve the given order. Handy for RBF.
```

RBY (Resolve by yourself)

Reply:
f1f60fb7ee11f098739da5f79d3d88e99fe842db